ultravox/data/datasets.py

```diff
@@ -258,14 +261,18 @@ def _load_audio_dataset(
         else:
             dataset = datasets.load_dataset(
                 path, name, split=split, trust_remote_code=True, streaming=streaming
-            ).cast_column("audio", datasets.Audio(sampling_rate=SAMPLE_RATE))
+            )
+            for column_name in self.BASE_AUDIO_COLUMNS:
+                dataset = dataset.cast_column(
+                    column_name, datasets.Audio(sampling_rate=SAMPLE_RATE)
+                )
```

Comment:
Bugfix for datasets that have audio column that is not named `audio`.

Reply:
This was not an issue for SODA since it was constructed with 16K Hz, but it was sloppy of me.